ubuntu-github-actions-android/Dockerfile-30.0.3

```diff
@@ -0,0 +1,71 @@
+# Version: 20231114
+# Image name: quay.io/opencv-ci/opencv-androidsdk-30
+# For 4.x branch
+
+FROM ubuntu:20.04
```

Comment:
Why no 22.04?

Reply:
I just used the same base image as before. It does not actually matters here, because most of work is done by cross tools.